### PR TITLE
KIALI-3048 Fix vertical scroll appearing in workload logs tab

### PR DESCRIPTION
**Pull request by israel-hdez**

### CODE REVIEW

**Comment by israel-hdez** [comment]
Issue was introduced in #1171 where the "title" was pulled out of the Info/overview tabs, reducing the space available for the logs pane. That PR went into v0.21.0 so we should backport.

**Comment by israel-hdez** [comment]
...this is a little brittle. Perhaps we should find a way to make it adapt regardless of the controls that are present. But that may involve more changes to the markup and css.

**Comment by josejulio** [comment]
> ...this is a little brittle. Perhaps we should find a way to make it adapt regardless of the controls that are present. But that may involve more changes to the markup and css.

Maybe with some flexbox

File: src/styles/_layout.scss [hunk 1]
```diff
@@ -6,9 +6,9 @@ body,
 .pf-c-page__main-section {
   /*
    * --pf-c-page__header--MinHeight is the height of the masthead
-   * 82px is the vertical space being used by the breadcrumb and tab headers
+   * 130px is the vertical space being used by the breadcrumb, tab headers, and the name of the {workload,svc,app}
    */
-  --kiali-details-pages-tab-content-height: calc(100vh - 82px - var(--pf-c-page__header--MinHeight));
+  --kiali-details-pages-tab-content-height: calc(100vh - 130px - var(--pf-c-page__header--MinHeight));
```
Comment (mtho11) [inline src/styles/_layout.scss]: Yeah, I wish we could get away from all of the pixel calculations.

**Comment by israel-hdez** [comment]
> Brittle yes, working yes.

Merging because it passes all checks :)

File: src/styles/_layout.scss (hunk 1 above)
Comment (israel-hdez) [reply src/styles/_layout.scss]: I think that the right way to make this happen is to contribute to patternfly-react. Anything we do in kiali-ui will involve hacking the PF css.

**Pull request merged**